mpas_analysis/default.cfg

```diff
@@ -3964,6 +3964,68 @@ volNH = PIOMAS/PIOMASvolume_monthly_climo_20180710.nc
 volSH = none
 
 
+[climatologyMapRiskIndexOutcomeNH]
+## options related to plotting maps of risk index outcomes for navigability
+## in sea-ice covered water from climatologies
+
+# colormap for model/observations
+colormapNameResult = RdYlBu
+# whether the colormap is indexed or continuous
+colormapTypeResult = indexed
+# color indices into colormapName for filled contours
+colormapIndicesResult = numpy.array(numpy.linspace(0, 255, 10), int)
+# colormap levels/values for contour boundaries
+colorbarLevelsResult = numpy.linspace(-10., 30., 9)
+
+# Months or seasons to plot (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['AMJ','OND']
+
+# comparison grid(s) on which to plot analysis
+comparisonGrids = ['arctic']
+
+# Polar Class of vessel according to IMO. Range is 1 to 12 (increments of 1).
+polarClass = 6
+
+# reference lat/lon for sea ice plots in the northern hemisphere
+minimumLatitude = 50
+referenceLongitude = 0
```

Comment:
It seems like the comment need to be updated, since you have a minimum, rather than a reference, latitude.